client/client.go

```diff
@@ -147,9 +147,8 @@ type Config struct {
 type Client struct {
 	requester Requester
 
-	maintenance      error
-	warningCount     int
```

Comment:
Maybe the count is important because, for example, the warning is the workload restarted, you want to know how many times it restarted in the last hour or minute because if it restarted like 100 times in the past minute, something is seriously wrong. I got this idea from Kubernetes where you can see the counts.

Reply:
The server can't return the count directly any more because the server side doesn't know which warnings are acknowledged anymore. However, if the client sees there's 1 or more warnings it could query `GET /v1/notices?types=warning` to get the number -- though I decided not to do that for now. If we want to add that later as warnings actually start to get used, it won't be hard.